Sources/NimiqClient/NimiqClient.swift

```diff
@@ -0,0 +1,1082 @@
+import Foundation
+
+// MARK: JSONRPC Models
```

Comment:
question: why did you choose to have all the code in one file? 

i think it would be easier on the eyes to split into multiple files.

Reply:
Thanks for your suggestion. I became accustomed to use the markers and the minimap for browsing the code with one big file, but I've been thinking about splitting the code in multiple files for all implementation clients in order to improve maintainability. In fact I will enable Codeclimate's maintainability analysis, to see what else could be improved in that regard.

Reply:
I split the code into these files [Account](https://github.com/rraallvv/swift-client/blob/master/Sources/NimiqClient/Models/Account.swift), [Block](https://github.com/rraallvv/swift-client/blob/master/Sources/NimiqClient/Models/Block.swift), [MemPool](https://github.com/rraallvv/swift-client/blob/master/Sources/NimiqClient/Models/MemPool.swift), [Miner](https://github.com/rraallvv/swift-client/blob/master/Sources/NimiqClient/Models/Miner.swift), [Node](https://github.com/rraallvv/swift-client/blob/master/Sources/NimiqClient/Models/Node.swift), [Peer](https://github.com/rraallvv/swift-client/blob/master/Sources/NimiqClient/Models/Peer.swift), and [Transaction](https://github.com/rraallvv/swift-client/blob/master/Sources/NimiqClient/Models/Transaction.swift). Each file contains structures, classes, enumerations, etc. related methods that handle the kind of data described by the file name. The maintainability analysis by CodeClimate is available [here](http://codeclimate.com/github/rraallvv/swift-client).